Public/Src/Engine/Processes/SandboxedProcessPipExecutor.cs

```diff
@@ -4022,13 +4033,24 @@ public async Task<bool> TryLogWarningAsync(SandboxedProcessOutput standardError,
                 return false;
             }
 
-            FormatOutputAndPaths(
+            bool errorWasTruncated = false;           
+            // Ignore empty lines
+            var standardErrorInResult = await standardError.ReadValueAsync();
+            var standardOutputInResult = await standardOutput.ReadValueAsync();
+            if (warningsError.Replace(Environment.NewLine, string.Empty).Trim().Length != standardErrorInResult.Replace(Environment.NewLine, string.Empty).Trim().Length ||
+                warningsOutput.Replace(Environment.NewLine, string.Empty).Trim().Length != standardOutputInResult.Replace(Environment.NewLine, string.Empty).Trim().Length)
+            {

```

Comment:
I think these are always going to be true as long as build output has anything else in addition to warnings.

Reply:
That's true. But that is what we have today. It will still eliminate the printing of the log path in some cases when we only have warning.